### ISSUE TITLE
Sairaaladatan 25.4. päivän tiedoissa virheitä

### ISSUE THREAD

**Issue by tnnmk** [HS-Datadesk/koronavirus-avoindata#76]
Sairaaladatassa 25.4. päivän tietoihin on merkitty väärä aikaleima 2020-04-24. Lisäksi kyseisen päivän koko Suomen tiedoista totalHospitalised ja inWard ovat väärin. (Pitäisi olla 193 ja 133, eikä 190 ja 130).

**Comment by tnnmk** [comment]
Myös tämän päivän 26.4. tiedot on merkitty eilisen aikaleimoin 2020-04-25.

**Comment by quarian** [comment]
Joo, näin ilmeisesti oli - pitäisi olla myös korjautunut tätä kirjoittaessa. Tarkistatko omalta puoleltasi @tnnmk

**Comment by tnnmk** [comment]
Aikaleimat vaikuttaisivat olevan nyt kunnossa.

Sen sijaan 25.4. päivän totalHospitalised ja inWard näyttävät olevan koko Suomen osalta edelleen yllä mainitulla tavalla väärin. Perustan tämän väitteen kahteen havaintoon:

1. THL:n julkaisemassa taulukossa [1] oli tuona päivänä luvut 193 ja 133 ainakin silloin kun itse tarkistin.
2. Nykyiset luvut 190 ja 130 eivät vastaa ERVA-kohtaisten lukujen summia, jotka ovat siis mainitut 193 ja 133.

(On toki mahdollista, että luvut olivat olleet THL:n julkaisemassa taulukossa aiemmin tuona päivänä väärin ja korjattiin myöhemmin tuona päivänä.)

Vertasin nyt muidenkin päivämäärien osalta koko Suomen lukuja ERVA-kohtaisten lukujen summiin ja löysin yhden toisenkin epätäsmäävyyden: 9.4. päivän totalHospitalised-arvo on koko Suomen osalta 244, mutta ERVA-kohtaisten arvojen summa on 238. Tämän osalta en tiedä, esiintyikö sama epätäsmäävyys myös THL:n taulukossa kyseisenä päivänä.

[1] https://thl.fi/fi/web/infektiotaudit-ja-rokotukset/ajankohtaista/ajankohtaista-koronaviruksesta-covid-19/tilannekatsaus-koronaviruksesta

**Comment by quarian** [comment]
Tutkitaan.
